src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -37,14 +37,16 @@ public UiManager(Logic logic) {
     public void start(Stage primaryStage) {
         logger.info("Starting UI...");
 
-        //Set the application icon.
+        //Set the application icon
```

Comment:
I think there should be a space between // and Set.

Reply:
agreed. just realized the original AB3 does not use spaces consistently, might be worth addressing this in a future pr?

Reply:
Yup, I'll add an issue!

Reply:
thanks! @vevek